explorer/client/src/components/JobRuns/TaskRuns.tsx

```diff
@@ -59,6 +62,15 @@ const TaskRuns = ({ etherscanHost, taskRuns, classes }: IProps) => {
                   {run.status}
                 </StatusIcon>
                 <Typography variant="body1">{run.type}</Typography>
+                {run.minimumConfirmations && (
```

Comment:
What do you feel about being more explicit about the condition here?

Reply:
Can you give me an example?